src/goto-instrument/accelerate/acceleration_utils.cpp

```diff
@@ -1070,7 +1069,7 @@ bool acceleration_utilst::assign_array(
   replace_expr(arr, fresh_array, changed);
 
   symbolt k_sym=fresh_symbol("polynomial::k", unsigned_poly_type());
-  exprt k=k_sym.symbol_expr();
+  symbol_exprt k = k_sym.symbol_expr();
 
```

Comment:
I would make that one const, which makes it clearer that 'k' later on doesn't change